src/terms.js

```diff
@@ -298,6 +299,7 @@ export default [
     'maximi(s|z)e',
     'meaningful( client | )interactions?',
     'measurements?',
+    'merge(er)?',
```

Comment:
Perhaps you meant this?

```suggestion
    'merger?',
```